python/lsst/ip/isr/isrTask.py

```diff
@@ -904,9 +904,9 @@ def readIsrData(self, dataRef, rawExposure):
             - ``bfKernel``: Brighter-Fatter kernel (`numpy.ndarray`)
             - ``defects``: list of defects (`lsst.meas.algorithms.Defects`)
             - ``fringes``: `lsst.pipe.base.Struct` with components:
-                - ``fringes``: fringe calibration frame (`afw.image.Exposure`)
-                - ``seed``: random seed derived from the ccdExposureId for random
-                    number generator (`uint32`)
+              - ``fringes``: fringe calibration frame (`afw.image.Exposure`)
+              - ``seed``: random seed derived from the ccdExposureId for random
+                  number generator (`uint32`)
```

Comment:
As above:
``seed``: `uint32`
Random seed derived from the ccdExposureId for number generator.